elixir/apps/domain/lib/domain/gateways.ex

```diff
@@ -419,30 +411,6 @@ defmodule Domain.Gateways do
     end
   end
 
-  def connect_gateway(%Gateway{} = gateway) do
-    with {:ok, _} <-
-           Presence.track(
-             self(),
-             group_gateways_presence_topic(gateway.group_id),
-             gateway.id,
-             %{}
-           ),
-         {:ok, _} <-
-           Presence.track(
-             self(),
-             account_gateways_presence_topic(gateway.account_id),
-             gateway.id,
-             %{
-               online_at: System.system_time(:second)
-             }
-           ) do
-      :ok = PubSub.subscribe(gateway_topic(gateway))
-      :ok = PubSub.subscribe(group_gateways_topic(gateway.group_id))
```

Comment:
The `group_gateways_topic` was only used for disconnecting all gateways in a group, but this is already handled via its token - deleting the token disconnects all entities listening on that socket_id, so this subscription was completely redundant.